### ISSUE TITLE
random_lax_test::LaxRandomTest.testBinomialSample is sensitive to its random seed

### ISSUE THREAD

**Issue by froystig** [google/jax#18941]
### Description

`testBinomialSample` appears to be sensitive to key changes. For example, several generated tests fail if we change the seed here to `1234567`:

https://github.com/google/jax/blob/b077483bfaaf197b79717a86bee3e626474e93f2/tests/random_lax_test.py#L1174-L1175

Similar to #18932.

### What jax/jaxlib version are you using?

0.4.22.dev20231211+b225c86f1 0.4.21

### Which accelerator(s) are you using?

CPU

### Additional system info?

numpy 1.26.2

**Comment by jakevdp** [comment]
In my experience a lot of these distribution tests are sensitive to a change in seed.

**Comment by jakevdp** [comment]
Agreed - but this is something that we need to look at more comprehensively in the random distribution test suite. I don't think the Binomial test is an outlier here!

**Comment by froystig** [comment]
I see what you mean and I agree. I've renamed the issue and tried to update the description to describe things better now.
